wikidatarefisland/pipes/statement_filters.py

```diff
@@ -0,0 +1,14 @@
+def referenced_statement_excluder(statement):
```

Comment:
I risk you not getting the pop culture reference, but what the heck...

![](https://media.giphy.com/media/Gu0KrktU8NRRu/giphy.gif)

I love seeing tightly organized functions, good code!

Reply:
Yeah but can this all be put into one class?